sdk/storage/azdatalake/sas/service.go

```diff
@@ -94,7 +95,8 @@ func (v DatalakeSignatureValues) SignWithSharedKey(sharedKeyCredential *SharedKe
 		string(v.Protocol),
 		v.Version,
 		resource,
-		"",                   //snapshot not supported
+		"", //snapshot not supported
+		v.EncryptionScope,
```

Comment:
This should be added for User Delegation SAS as well in the method `SignWithUserDelegation`.